css/slashed-components.css

```
@@ -3229,4 +3305,93 @@
+       <div class="cs-floating-label">
+         <input type="text" id="f" placeholder=" ">
+         <label for="f">Email address</label>
+       </div> */
+
+  .cs-floating-label { position: relative; }
+  .cs-floating-label label {
+    position: absolute;
+    inset-block-start: 50%;
+    inset-inline-start: var(--space-s);
+    translate: 0 -50%;
+    transform-origin: inline-start top;
```

Stylelint (17.9.0)

[error] Unknown value "inline-start" for property "transform-origin" (declaration-property-value-no-unknown)

(declaration-property-value-no-unknown)